envelope.go

```diff
@@ -26,8 +27,8 @@ type Envelope struct {
 	// All non-text parts that were not placed in Attachments or Inlines, such as multipart/related
 	// content.
 	OtherParts []*Part
-	Errors     []*Error              // Errors encountered while parsing
-	header     *textproto.MIMEHeader // Header from original message
+	Errors     []*Error          // Errors encountered while parsing
+	header     *nettp.MIMEHeader // Header from original message
```

Comment:
I think this would be cleaner for people looking at docs if you rename the import of `.../internal/textproto` instead of `net/textproto`.